src/services/mock/checkpoint/get_checkpoint.py

```diff
@@ -28,6 +32,14 @@
                 submitNum
             }
         }
+        familyProxies (sort: { keys: ["firstParent"]}) {
+            id
```

Comment:
Might be able to replace `id` with `name` and `cyclePoint` to avoid parsing?

Reply:
Done!